tests/dependencies_test.go

```diff
@@ -24,7 +24,9 @@ func TestDependenciesLifecycle(t *testing.T) {
 
 	upgradeWordpressBundle(p, namespace)
 
-	invokeWordpressBundle(p, namespace)
+	upgradeWordpressBundle(p, namespace)
```

Comment:
Should we remove this extra `upgradeWordpressBundle(p, namespace)` invocation?

Reply:
Should we create a TODO to uncomment `invokeWordpressBundle(p, namespace)` ?  And/or add a comment mentioning what is currently preventing a successful test?

Reply:
Oops! Thanks, I commented that out when I was debugging something and forgot to put it back in. Fixing....

Reply:
Same for the extra upgrade call, I just removed that and pushed up the fix.